main/mains.py

```diff
@@ -0,0 +1,87 @@
+import os
+import sys
+import inspect
+currentdir = os.path.dirname(os.path.abspath(
+    inspect.getfile(inspect.currentframe())))
+parentdir = os.path.dirname(currentdir)
+sys.path.insert(0, parentdir)
+
+from model.rooms import Office
+from model.rooms import LivingSpace
+from model.person import Fellow
+from model.person import Staff
+import random
+
+
+class Amity(object):
+    def __init__(self):
+        self.people = []
+        self.offices = []
+        self.living_spaces = []
+        self.unallocated = []
+        self.unallocated_fellows = []
+
+    def populate(self):
+        """prepopulates amity with 10 offices and 10 livingspaces"""
+        office_names = ['Valhalla', 'Oculus', 'Krypton', 'Shire',
+                        'Hogwarts', 'Mordor', 'Orange', 'Turquoise',
+                        'Peach', 'Cyan']
+
+        for office in range(0, len(office_names)):
```

Comment:
**Code Quality: Nitpick**

We could also write this as:

```py
for office_name in office_names:
    self.offices.append(Office(office_name))
```

Without the range operation. It is much easier to read IMO and gets rid of the indexes.

The same thing was done below.